src/components/molecules/Medias/Medias.tsx

```diff
@@ -0,0 +1,76 @@
+import { Discord } from "src/components/atoms/icons/Discord/Discord";
+import { Facebook } from "src/components/atoms/icons/Facebook/Facebook";
+import { Forums } from "src/components/atoms/icons/Forums/Forums";
+import { Instagram } from "src/components/atoms/icons/Instagram/Instagram";
+import { Tumblr } from "src/components/atoms/icons/Tumblr/Tumblr";
+import { Twitch } from "src/components/atoms/icons/Twitch/Twitch";
+import { Twitter } from "src/components/atoms/icons/Twitter/Twitter";
+import { Youtube } from "src/components/atoms/icons/Youtube/Youtube";
+
+enum ICON_SIZE {
+  small = "h-7 w-7",
+}
```

Comment:
Just need to take care with something like that because if you don't use this on className and in any other place in the app tailwind couldn't compile this class.
A better approach should be to create a component to abstract the size and pass just size="small"